content/tutorials/spatial_standards_raster/index.Rmd

```diff
@@ -0,0 +1,227 @@
+---
+title: "How to use the GeoTIFF raster file format in R"
+description: "A simple tutorial to demonstrate the use of GeoTIFF files in R"
+author: "Floris Vanderhaeghe"
+date: 2019-11-20
+categories: ["r", "gis"]
+tags: ["gis", "r"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE
+)
+set.seed(123456789)
+unlink("art*")
+```
+
+
+This tutorial uses a few basic functions from the [dplyr](https://dplyr.tidyverse.org) and [raster](https://rspatial.org/raster/pkg) packages.
+While only a few functions are used, you can use the previous hyperlinks to access the tutorials (vignettes) of these packages for more functions and information.
+
+```{r warning=FALSE, message=FALSE}
+options(stringsAsFactors = FALSE)
+library(raster)
+library(tidyverse)
+library(inborutils)
+```
+
+You will find a bit more background about 'why and what', regarding the open GeoTIFF standard, in [a separate post](../../articles/geospatial_standards/) on this website.
+
+In short, the GeoTIFF format is ideal for exchange, publication, interoperability & durability and to open science in general.
+
+## How to make and use GeoTIFF files (`*.tif`)
+
+### Making a mono-layered GeoTIFF file from a `RasterLayer` R object
+
+Let's create a small dummy `RasterLayer` object from scratch, for some area in Belgium (using the CRS ^[CRS = coordinate reference system] Belgian Lambert 72, i.e. [EPSG-code 31370](https://epsg.io/31370)):
+
+```{r}
+artwork <- 
+  raster(extent(188500, 190350, 227550, 229550), # xmin, xmax, ymin, ymax
+         res = 50, # resolution of 50 meters
+         crs = CRS("+init=epsg:31370")) %>% 
+  setValues(runif(ncell(.)))  # fill with random values
+```
+
+What does this look like?
+
+```{r}
+artwork
+```
+
+A simple trick to plot this raster:
+
+```{r artwork}
+spplot(artwork)
+```
+
+To write this `RasterLayer` object as a GeoTIFF, you can use the `raster::writeRaster()` function.
+In the background, it uses the GeoTIFF driver of the powerful [GDAL](https://gdal.org) library.
+
+```{r}
+artwork %>% 
+  writeRaster("artwork.tif")
+```
+
+And now?\
+_**Say HURRAY!!**_
+
+### Making a multi-layered GeoTIFF file from a `RasterBrick` R object
+
+Let's create a `RasterBrick` object of three layers:
+
+```{r}
+arts <- brick(artwork) # RasterBrick with one layer (the RasterLayer from above)
+arts[[2]] <- artwork + 10 # Add second layer, e.g. based on first one
+arts[[3]] <- calc(arts[[2]], function(x) 20 ^ x) # Making third layer from second
+names(arts) <- paste0("layer", 1:3)
+```
+
+_Note: in real-life cases, you will see that the `calc()` function is more efficient than simple algebraic expressions such as for layer2._
+
+How does the result look like?
+
+```{r}
+arts
+```
+
+```{r arts}
+arts %>% 
+  as.list %>% 
+  lapply(spplot)
+```
+
+So now what?\
+_**Let's write it!**_
+
+```{r}
+arts %>% 
```

Comment:
piping seems a bit overkill in these simple cases

Reply:
A matter of taste, I do like this more.